src/cswinrt/code_writers.h

```diff
@@ -7640,26 +7653,203 @@ return false;
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        get => (nint)Unsafe.AsPointer(in Vftbl);
+    }
+
+    [UnmanagedCallersOnly(CallConvs = [typeof(CallConvMemberFunction)])]
+    private static int Invoke(%)
+    {
+        %
+    }
+}
+)",
+            type.TypeName(),
+            type.TypeName(),
+            type.TypeName(),
+            bind<write_abi_parameters>(signature),
+            bind<write_managed_method_call>(
+                signature,
+                w.write_temp("ComInterfaceDispatch.GetInstance<%>((ComInterfaceDispatch*)thisPtr).%%",
+                    bind<write_type_name>(type, typedef_name_type::Projected, false),
+                    method.Name(),
+                    "(%)"),
+                false)
+        );
+
+    }
+    void write_delegate_comwrappers_callback(writer& w, TypeDef const& type)
+    {
+        w.write(R"(
+file abstract unsafe class %ComWrappersCallback : IWindowsRuntimeObjectComWrappersCallback
+{
+    /// <inheritdoc/>
+    public static object CreateObject(void* value, out CreatedWrapperFlags wrapperFlags)
+    {
+        WindowsRuntimeObjectReference valueReference = WindowsRuntimeMarshal.CreateObjectReferenceUnsafe(
+            externalComObject: value,
+            iid: in %ReferenceImpl.IID,
+            wrapperFlags: out wrapperFlags);
+
+        return new %(valueReference.Invoke);
+    }
+}
+)",
+            type.TypeName(),
+            type.TypeName(),
+            bind<write_type_name>(type, typedef_name_type::Projected, false)
+        );
+    }
+
+
+    void write_delegate_com_wrappers_marshaller_attribute_impl(writer& w, TypeDef const& type)
+    {
+        auto name = type.TypeName();
+        w.write(
+            R"(internal sealed unsafe class %ComWrappersMarshallerAttribute : WindowsRuntimeComWrappersMarshallerAttribute
+{
+    public override void* GetOrCreateComInterfaceForObject(object value)
+    {
+        return WindowsRuntimeMarshal.GetOrCreateComInterfaceForObject(value, CreateComInterfaceFlags.TrackerSupport);
+    }
+
+    public override ComInterfaceEntry* ComputeVtables(out int count)
+    {
+        count = sizeof(ReferenceInterfaceEntries) / sizeof(ComInterfaceEntry);
+        return (ComInterfaceEntry*)Unsafe.AsPointer(in %InterfaceEntriesImpl.Entries);
+    }
+}
+
+)", name, name);
+    }
+
+    void write_delegate_marshaller(writer& w, TypeDef const& type)
+    {
+        if (is_exclusive_to(type))
+        {
+            return;
+        }
+
+        auto projected_type = w.write_temp("%", bind<write_type_name>(type, typedef_name_type::Projected, false));
+        w.write(R"(
+[global::System.ComponentModel.EditorBrowsable(global::System.ComponentModel.EditorBrowsableState.Never)]
+public static unsafe class %Marshaller
+{
+    public static WindowsRuntimeObjectReferenceValue ConvertToUnmanaged(% value)
+    {
+        return WindowsRuntimeDelegateMarshaller.ConvertToUnmanaged(value, in %ReferenceImpl.IID);
+    }
+    #nullable enable
```

Comment:
Why only here? Technically speaking `ConvertToUnmanaged` also can receive a `null` value. But also shouldn't we enable nullability annotations in the whole file? We can also do that as a follow up. We could also skip it given this is all private implementation detail code anyway. @manodasanw thoughts?

Reply:
<img width="1653" height="751" alt="image" src="https://github.com/user-attachments/assets/37f8ec4d-8d2b-4b45-8664-2c8fcd1ab927" />
I was getting these errors without so thought we needed it here

Reply:
Thought doing this to the whole file would be overkill and will hide compile issues that we don't want it to hide. So I made it targeted to this method. The other one didn't have this issue. 

`The annotation for nullable reference types should only be used in code within a '#nullable' annotations context. Auto-generated code requires an explicit '#nullable' directive in source.`

Reply:
Lets hold off on doing it to the entire file and just do scoped for now.  I was looking at if we should do this for the entire generated file, but then that brings the question about what things in the generated projections should be nullable or not.  So, I was going to look at that later after we got everything else working.

Reply:
My idea is to enable in the entire file and disable it over projected APIs, since those are nullable-unaware.